universal-login-relayer/lib/core/models/RepositoryItem.ts

```diff
@@ -0,0 +1,7 @@
+import {MessageState} from '@universal-login/commons';
+
+export interface RepositoryItem {
```

Comment:
The naming is tricky. `RepositoryItem` talks about role in the architecture, but not in domain, which is a bad naming. 
Should be something like `Minable` or `Exectuable`. I like `Mineable`

Reply:
For Type Safety I would do sth like:
```ts
type Minable = Message | Deployment;
```

Reply:
Renamed `RepositoryItem` to `Mineable`